.autogit/autogit.json

```diff
@@ -0,0 +1,5 @@
+{
+  "updateInterval": 1800,
+  "logging": true,
+  "silent": true
+}
```

Comment:
What is this file for? I think we should not include this in the template repository. Would be great to understand how this helps the user.